tests/test_run_service.py

```
@@ -435,12 +462,14 @@ def test_list_runs_paginated_sorts_newest_first(self, tmp_path, monkeypatch):
 
         # Mock storage functions to use only our test runs
         monkeypatch.setattr(storage, "list_runs", lambda runs_dir=None: run_ids)
+        monkeypatch.setattr(storage, "scan_runs", lambda runs_dir=None: (run_ids, []))
```

Ruff (0.14.13)

Unused lambda argument: `runs_dir`

(ARG005)